tools/scripts/setup-coral-credits.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -eux
+
+SITE=https://credits.apps.staging.hpc.cam.ac.uk
+
+TEST_PASSWORD="testpassword"
+
+# Set up some variables
+CONTENT_TYPE="Content-Type: application/json"
+
+# Get a token
+echo "Getting an auth token:"
+TOKEN=$(curl -s -X POST -H "$CONTENT_TYPE" -d \
+    "{
+        \"username\": \"admin\",
+        \"password\": \"$TEST_PASSWORD\"
+    }" \
+    $SITE/api-token-auth/ | jq -r '.token')
+echo "Auth Token: $TOKEN"
+
+AUTH_HEADER="Authorization: Bearer $TOKEN"
+
+# 1. Add a resource provider
+echo "Adding a resource provider:"
+RESOURCE_PROVIDER_ID=$(curl -s -X POST -H "$AUTH_HEADER" -H "$CONTENT_TYPE" -d \
+    '{
+        "name": "Test Provider",
+        "email": "provider@test.com",
+        "info_url": "http://testprovider.com"
+    }' \
+    $SITE/resource_provider/ | jq -r '.url')
+echo "Resource Provider URL: $RESOURCE_PROVIDER_ID"
+
+# 2. Add resource classes
+echo "Adding resource classes:"
+VCPU_ID=$(curl -s -X POST -H "$AUTH_HEADER" -H "$CONTENT_TYPE" -d '{"name": "VCPU"}' $SITE/resource_class/ | jq -r '.id')
+MEMORY_ID=$(curl -s -X POST -H "$AUTH_HEADER" -H "$CONTENT_TYPE" -d '{"name": "MEMORY_MB"}' $SITE/resource_class/ | jq -r '.id')
+DISK_ID=$(curl -s -X POST -H "$AUTH_HEADER" -H "$CONTENT_TYPE" -d '{"name": "DISK_GB"}' $SITE/resource_class/ | jq -r '.id')
+echo "Resource Class IDs: VCPU=$VCPU_ID, MEMORY_MB=$MEMORY_ID, DISK_GB=$DISK_ID"
+
+# 3. Add an account
```

Comment:
Should we drop these bits from this script, and leave that to the other script that adds credits for a given account?